### PR TITLE
Postgres built through docker 

### PR DESCRIPTION
**Pull request by aneesh4995**

### CODE REVIEW

**Comment by aneesh4995** [comment]
done with dockerizing postgres @rgavinc @yashShelatkar

File: README.md [hunk 1]
```diff
@@ -47,5 +47,16 @@ To get started, fork the repository and run the following commands:
 
 To get started clone the drum-root-api and run the following commands
 
-    docker build -t drum-root-api . (NOTE .(dot) at last is important)
-    docker run -it -p 3000:3000 drum-root-api
+    sudo docker-compose up --build (Gets running on port 3000)
+
+To access the PSQL shell through local
+
+    docker exec -it 05b3a3471f6f bash
```
Comment (rgavinc) [inline README.md]: This command isn't working for me after running `docker-compose up --build`,  I get:
`No such container: 05b3a3471f6f`
- [ ] Make sure that is the correct command.

**Review by rgavinc — CHANGES_REQUESTED** [review]
Looking good, I'm just not sure that the readme correctly covers how to access the DB.

**Comment by aneesh4995** [comment]
Done with the changes. Please test and revert any updations required @rgavinc

**Review by rgavinc — APPROVED** [review]
Awesome work!

**Pull request merged**